schema.sql

```diff
@@ -6,5 +6,6 @@ CREATE TABLE animals (
     date_of_birth DATE NOT NULL,
     escape_attempts INTEGER NOT NULL,
     neutered BOOLEAN NOT NULL,
-    weight_kg DECIMAL NOT NULL
+    weight_kg DECIMAL NOT NULL,
+    species VARCHAR(50)  NULL
```

Comment:
- [ ] **[Optional:]** You can also try the following command to add a new column like "species". 
`ALTER TABLE animals ADD species varchar(255);`